packages/pages/src/vite-plugin/build/buildStart/buildStart.ts

```diff
@@ -60,18 +61,26 @@ const copyPluginFiles = (fileEmitter: EmitFile) => {
     startLog: "Copying Yext plugin files",
   });
 
-  const currentPath = new URL(import.meta.url).pathname;
-  const pathToPluginsDir = path.resolve(currentPath, "../../../../plugin");
-  const pluginFiles = glob.sync(`${pathToPluginsDir}/*.ts`);
+  const currentPath = fileURLToPath(import.meta.url);
+  const pathToPluginsDir = path.resolve(
+    currentPath,
+    path.join("..", "..", "..", "..", "plugin")
```

Comment:
Pretty sure this could be one string and it's replace the slashes automatically.

Reply:
that feels more confusing than just keeping it like this (bc it mutates the "/")